README.md

```diff
@@ -145,8 +142,14 @@ Compile the library. Files are compiled into the `./dist` folder.
 npm run build
 ```
 
-Follow the instructions from the point [Create self signed SSL certificate](#create-self-signed-ssl-certificate).
-(Only `path` in command `http-server` will be `./examples/`.)
+Create certificate as it is described in [Create self signed SSL certificate](#create-self-signed-ssl-certificate). Create symlinks to the distributed library and node modules (only once) and run server:
+
+```
+ln -s ../node_modules examples/node_modules
```

Comment:
Is there really need these links? From the doc https://nodejs.org/api/modules.html#modules_loading_from_node_modules_folders looks like nodejs is able to find node_modules also in parent folder, it doesn't?
Also, dist folder could be removed by some additions in webpack config, couldn't?